src/mlpack/methods/ann/activation_functions/mish_function.hpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file mish_function.hpp
+ * @author Kartik Dutt
+ *
+ * Definition and implementation of the mish function as described by
+ * Diganta Misra.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @misc{
+ *   author = {Diganta Misra},
+ *   title = {Mish: Self Regularized Non-Monotonic Neural Activation Function},
+ *   year = {2019}
+ * }
+ * @endcode
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_MISH_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_MISH_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <algorithm>
+#include <mlpack/methods/ann/activation_functions/softplus_function.hpp>
+
+namespace mlpack
```

Comment:
For namespaces we usually write:  `namespace mlpack {`, so the opening `{` in the same line.

Reply:
Sorry about, I have made the required changes for all the above points. Kindly have a look,
Thanks.

Reply:
Again, no worries, it's easy to miss those kinds of things.